content/en/storage-provider/videos/actors.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "How Filecoin Actors Work"
+description: "Overview of Filecoin Actors"
+menu:
+    storageprovider:
+        parent: "storage-provider-videos"
+---
+
+This talk given by zenground0 in July of 2022 takes a deep dive into the actors that make up the computational model of the Filecoin protocol. Learn about why Filecoin uses actors, the different types of actors, and what the current eleven available Filecoin actors do. Understand the upgrades that have occurred since mainnet launch. As the Filecoin Virtual Machine (FVM) is developed, the capabilities of Filecoin actors will greatly expand along with the potential for the Filecoin Protocol, by decoupling adding functionality from protocol upgrades and allowing for independent user actors.
+
+<iframe width="708" height="398" src="https://www.youtube.com/embed/9JbwbTPonv0" title="How Filecoin Actors Work" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [What are Filecoin actors](#what-are-filecoin-actors)
+- [The Actor Model and Blockchain Consensus](#the-actor-model-and-blockchain-consensus)
+- [Actor Code](#actor-code)
+- [Filecoin is complicated](#filecoin-is-complicated)
+- [Future Actors and Two Layers](#future-actors-and-two-layers)
+- [The FVM’s Impact](#the-fvms-impact)
+- [Verifiable Storage](#verifiable-storage)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
The TOC gets auto generated on each page already. Don't think this is necessary:

![image](https://user-images.githubusercontent.com/9611008/193632492-518b1a24-59e9-40bf-8740-f90dbb4896c1.png)

Reply:
Removed the TOC.